client/me/purchases/purchases-site/style.scss

```diff
@@ -45,3 +45,11 @@
 		padding: 11px 24px;
 	}
 }
+
+@include breakpoint( ">960px" ) {
```

Comment:
Breakpoint should be specific to the class instead of the other way around.

Reply:
I don't think it matters either way, but I updated it for consistency. Thanks.